### PR TITLE
docs: update `getting-started` guide

### LINKED ISSUE #103: Docs: Add steps to create issue and make screenshot of profile on the README
## Background

- We want contributors to learn how to create an issue as part of contributing flow.
- To prevent mistakes of not running the command to generate the profile, we want to ensure contributors providing a screenshot of their profile on the README after being generated to be included in their PRs.

## Suggested solution

In the [Getting Started](https://intro.opensauced.pizza/#/05-how-to-contribute-to-open-source?id=getting-started) section:

- [x] Add step 1 to create an issue.

> 1. Create an issue by clicking the "New issue" button in the "Issues" tab and complete the form.

- [x] Add a step to make a screenshot right after the step to "Run `npm run contributors:generate` in your terminal to generate the guest book on the README.".

> X. Copy and paste the Markdown of the README in [Markdown Live Preview](https://markdownlivepreview.com/) and take a screenshot of your profile being generated. You will need this later when creating a pull request. **Tips**: If you don't see your profile in the "Contributors" section, zoom out the screen until you can see it before taking a screenshot.
  
- [x] Fix the numbering of the steps accordingly.
- [x] Check the code and the live preview format of the section. You can use [Markdown Live Preview](https://markdownlivepreview.com/) to live preview the Markdown. 

Below is the code block of the "Getting Started" section:

https://github.com/open-sauced/intro/blob/a5a10bf39a6c32bc56868aaf973f3ddd60881be6/05-how-to-contribute-to-open-source.md?plain=1#L140-L239

### PR DESCRIPTION
**Pull request by at-the-vr**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents
Closes #103 

## Steps to QA

1. Open the PR in GitPod
2. Use Live Server or Markdown Live Preview extension to see the suggested changes
3. Validate the changes, find any discrepancies and provide suggestions

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed
- [x] 📝 [05-how-to-contribute-to-open-source](https://intro.opensauced.pizza/#/05-how-to-contribute-to-open-source?id=getting-started)

### CODE REVIEW

**Review by BekahHW — CHANGES_REQUESTED** [review]
Hey @at-the-vr! Thanks for taking on this issue. Would you mind removing the extra space after line seven and making guest book one word? We realized in reviewing your PR that we had it written both ways, but we're going to make it consistent throughout the repo with one word. Thanks so much for getting in this PR! Everything else is great.

File: 05-how-to-contribute-to-open-source.md [hunk 1]
```diff
@@ -208,9 +210,13 @@ You'll need to have these tools donwloaded and installed on your local machine:
 
        > ⚠️ **Remember to add a comma after the quotes, except for the last one in the array and the last closing curly bracket.**
 
-6. Run `npm run contributors:generate` in your terminal to generate the guest book on the README.
+7. Run `npm run contributors:generate` in your terminal to generate the guest book on the README.
+
```
Comment (BekahHW) [inline 05-how-to-contribute-to-open-source.md:214]: Can you remove this space?

File: 05-how-to-contribute-to-open-source.md [hunk 2]
```diff
@@ -139,11 +139,13 @@ You'll need to have these tools donwloaded and installed on your local machine:
 
 ### Getting Started
 
-1. Fork the [guest book repository](https://github.com/open-sauced/guestbook).
-2. Clone your forked repository to your computer.
-3. Run `npm install` to install the dependencies.
-4. Create a new branch and use a descriptive name related to your contribution — for example, `feat/add-alice`.
-5. Add yourself to the guest book.
+1. Create an issue by clicking the "New issue" button in the "Issues" tab and complete the form.
+2. Fork the [guest book repository](https://github.com/open-sauced/guestbook).
```
[the remark this answers is not in the document]
Comment (BekahHW) [reply 05-how-to-contribute-to-open-source.md:143]: And also remove the space between lines 2 and 3.

**Review by CBID2 — APPROVED** [review]
Looks ok to me! :) What do you think @BekahHW?

**Review by BekahHW — APPROVED** [review]

**Comment by BekahHW** [comment]
Great job, @at-the-vr !

**Pull request merged**